common/logisticspipes/gui/guidebook/book/MenuItem.java

```diff
@@ -0,0 +1,127 @@
+package logisticspipes.gui.guidebook.book;
+
+import java.util.ArrayList;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.audio.PositionedSoundRecord;
+import net.minecraft.client.audio.SoundHandler;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.renderer.RenderHelper;
+import net.minecraft.init.SoundEvents;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+
+import lombok.Getter;
+
+import logisticspipes.LPConstants;
+import logisticspipes.gui.guidebook.GuiGuideBook;
+import logisticspipes.utils.GuideBookContents;
+import logisticspipes.utils.string.StringUtils;
+
+public class MenuItem {
+
+	private static final ResourceLocation GUI_BOOK_TEXTURE = new ResourceLocation(LPConstants.LP_MOD_ID, "textures/gui/guide_book.png");
+
+	// Getting constants
+	private final int z$titleButtons = 15; // Title and Buttons Z
+	private final int z$frame = 10; // Frame Z
+	private final int z$text = 5;// Text/Information Z
+	private final int z$background = 0;  // Background Z
+
+	//// Information storage
+	@Getter
+	private GuideBookContents.Chapter chapter;
+
+	//// Drawing variables
+	public boolean visiible, hovering, enabled;
```

Comment:
`visiible` → `visible`

Reply:
Yeah actually spotted that one while resolving the previous but thanks hahaha